### ISSUE TITLE
Previews for 10bit H264/AVC videos are displayed with wrong colors

### ISSUE THREAD

**Issue by Meru** [linuxmint/nemo#2129]
```
 * Nemo  4.0.6
 * Is issue with: Windowed
 * Distribution: Mint 19.1
 * Graphics hardware: not relevant
 * 64bit
 ```

**Issue**
Previews for 10bit H264/AVC videos are displayed with wrong colors:
https://i.imgur.com/TtBlNwC.png

**Steps to reproduce**
Just browse to the file in question, switch to Icon View to see better.

**Expected behaviour**
Colors should be correct (not everything purple!)

**Other information**
This may be not a problem with nemo per se, but not sure where else I can put this.

It looks like 2 extra bits got clamped at some point during conversion. Interestingly, this does not happen with 10bit h265/HEVC files.
switching to ffmpegthumbnailer as generator produces correct results (but also draws an ugly frame around previews, win7-style).

**Comment by leigh123linux** [comment]
Do you have totem installed?, looks like their version is too old to include.

https://github.com/GNOME/totem/commit/621a387cbe88c8084796dd601e1efc95b6fd2c00#diff-cbb037b910e14cf18ad8df0b9457c994

https://packages.ubuntu.com/search?keywords=totem

I don't think xplayer has the thumbnail issue as it was patched

https://github.com/linuxmint/xplayer/commit/d8fec4bae377da16ffed3518d73a8e11f8533b99#diff-ec095fcc14ed88eda9cf68c6c3f15fc2

**Comment by Meru** [comment]
no, i only have whatever Mint 19.1 has by default + mpv that has no effect on any of that whatsoewver. My system is mostly clean and i'm certain it behaves the same on latest liveDVD

**Comment by shahmajlani** [comment]
@Meru, you can remove the "movie strip" effect of ```ffmpegthumbnailer``` by removing the "-f" argument on the thumbnailer service launcher thingy. 
on my system it's located at ```/usr/share/thumbnailers/ffmpegthumbnailer.thumbnailer```

of course, this is for the workaround and issue probably stands (i haven't tested).

**Comment by Meru** [comment]
So I found out how to test xplayer's thumbnailer:
`$ xplayer-video-thumbnailer input output.png` results in a wring output, so this have to be an xplayer issue. You should be able to transfer this issue to the xplayer repo.

**Comment by RafaMartorell** [comment]
I solve this annoying problem
deleting the file xplayer.thumbnailer from the folder /usr/share/thumbnailers
and after this install totem
sudo apt install totem*
Curiously, with Totem installed, the thumbnails have worked again.
